geopandas/_config.py

```diff
@@ -91,32 +91,6 @@ def _validate_bool(value):
         raise TypeError("Expected bool value, got {0}".format(type(value)))
 
 
-def _default_use_pygeos():
-    import geopandas._compat as compat
-
-    return compat.USE_PYGEOS
-
-
-def _callback_use_pygeos(key, value):
-    assert key == "use_pygeos"
-    import geopandas._compat as compat
-
-    compat.set_use_pygeos(value)
-
-
-use_pygeos = Option(
-    key="use_pygeos",
-    default_value=_default_use_pygeos(),
-    doc=(
-        "Whether to use PyGEOS to speed up spatial operations. The default is True "
-        "if PyGEOS is installed, and follows the USE_PYGEOS environment variable "
-        "if set."
-    ),
-    validator=_validate_bool,
-    callback=_callback_use_pygeos,
-)
```

Comment:
I think we will need to keep this option for now, but just as a no-op (and then we can first deprecate it when someone uses it, before actually removing it)?